### PR TITLE
feat(bodies): add task description to bodies

### PR DESCRIPTION
**Pull request by LeonVreling**
Adds a new field "task description" to all bodies. This field is used to give a description of the work to be done in the statutory positions

### CODE REVIEW

**Comment by LeonVreling** [comment]
Should be merged together with https://github.com/AEGEE/core/pull/79

File: src/views/core/bodies/Single.vue [hunk 1]
```diff
@@ -93,10 +93,13 @@
                 </tr>
                 <tr>
                   <th>Description</th>
+                  <td>{{ body.description }}</td>
```
[the remark this answers is not in the document]
Comment (LeonVreling) [reply src/views/core/bodies/Single.vue:96]: Uhm yeah I can add markdown there again, but I believe they didn't have it before as well. If I add it, then the bodies list should also be updated to use the markdown, as currently it is not.

**Comment by LeonVreling** [comment]
> One thing I was missing though is filtering on the type of body, because now locals will also have a task description. So maybe you can check a box on the body editing page if they would like to add a task description? And then if the task description is empty it is not shown. Would that be possible?

What do you think about only showing the field on the body page when there is a task description set, that should be really easy to do

**Comment by LeonVreling** [comment]
>One thing I just thought of is that it might be better not to link it to bodies but to circles, than we can also have different task descriptions for the various CD members (for example)

I don't really think that is necessary for now, but maybe once we have the Gsuite wrapper and we redo the circles, that would be nice to also have

**Comment by WikiRik** [comment]
> > One thing I was missing though is filtering on the type of body, because now locals will also have a task description. So maybe you can check a box on the body editing page if they would like to add a task description? And then if the task description is empty it is not shown. Would that be possible?
> 
> What do you think about only showing the field on the body page when there is a task description set, that should be really easy to do
> 

I think that would still confuse the board members, but if you include a tooltip on the Edit page explaining that it is not applicable for locals then I am fine with that.

**Comment by WikiRik** [comment]
> >One thing I just thought of is that it might be better not to link it to bodies but to circles, than we can also have different task descriptions for the various CD members (for example)
> 
> I don't really think that is necessary for now, but maybe once we have the Gsuite wrapper and we redo the circles, that would be nice to also have

I can stand behind that. I will add it to Jira as low priority. Would not be a big change so that's okay

File: src/views/core/bodies/Single.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (WikiRik) [reply src/views/core/bodies/Single.vue:96]: I don't agree that the bodies list should support markdown. That will only made the list a lot longer than is needed. But consistency is key so I withdraw this comment

File: src/views/core/bodies/Single.vue [hunk 2]
```diff
@@ -89,14 +89,19 @@
...
                 <tr>
                   <th>Title</th>
-                  <td>{{ body.name}}</td>
+                  <td>{{ body.name }}</td>
                 </tr>
                 <tr>
                   <th>Description</th>
                   <td>
-                    <span v-html="$options.filters.markdown(body.description)"></span>
+                    <span v-html="$options.filters.markdown(body.description)"/>
```
Comment (WikiRik) [inline src/views/core/bodies/Single.vue:97]: So we're keeping markdown in this?

File: src/views/core/bodies/Single.vue [hunk 3]
```diff
@@ -89,14 +89,19 @@
...
                   </td>
                 </tr>
+                <tr v-if="body.task_description !== null">
```
Comment (WikiRik) [inline src/views/core/bodies/Single.vue:100]: You tested this locally right?

Comment (LeonVreling) [reply src/views/core/bodies/Single.vue:100]: Yes I did, and it worked as supposed

File: src/views/core/bodies/Single.vue (hunk 2 above)
Comment (LeonVreling) [reply src/views/core/bodies/Single.vue:97]: Yes for consistency, it might be nice to be there on the page of the body itself I thought

**Pull request merged**